src/components/Select/Select.js

```diff
@@ -1,22 +1,70 @@
-import React from 'react'
+import React, { useState, useEffect } from 'react'
 import PropTypes from 'prop-types'
 // See React-Select -- https://github.com/JedWatson/react-select for documentation
 // on usage, Async configuration, etc.
 import ReactSelect from 'react-select'
 import ReactSelectAsync from 'react-select/async'
 import ReactSelectAsyncCreatable from 'react-select/async-creatable'
 import ReactSelectCreatable from 'react-select/creatable'
+import useErrorMessage from '../../hooks/useErrorMessage.js'
+import { InputLabel } from '../InputLabel'
 
 import styles from './Select.module.scss'
 
-export const Select = ({ className, title, isAsync, isCreatable, ...rest }) => {
+export const Select = ({
+  className,
+  title,
+  isAsync,
+  isCreatable,
+  validator,
+  onChange,
+  formChangeHandler,
+  currentError,
+  formTouched,
+  labelCopy,
+  name,
+  ...rest
+}) => {
+  const onChangeHandler = (event) => {
+    updateSelectedValue(event.value)
+
+    if (onChange) {
+      onChange(event)
+    }
+  }
+
+  const resolvedValidator = validator ? validator : () => ''
+  const [getError, setError, , validate] = useErrorMessage(resolvedValidator)
+  const [selectedValue, updateSelectedValue] = useState(undefined)
+
+  const validationSelect = () => {
+    const errorMessage = validate(selectedValue)
+    setError(errorMessage)
+    if (formChangeHandler) {
+      formChangeHandler(selectedValue, errorMessage)
+    }
+  }
+
+  useEffect(() => {
+    const isSelectedValue = typeof selectedValue !== 'undefined'
+    if (isSelectedValue) {
+      validationSelect()
+    }
+  }, [selectedValue])
+
+  const onBlur = () => {
+    validationSelect()
+  }
+
+  const wrapperClass = title ? styles.wrapper : ''
```

Comment:
How is title tied to the wrapper styles

Reply:
This line of code wasn't something I added. I simply moved it. So I'm not sure...